markdown/events/2022/hackathon-october-2022.md

```diff
@@ -44,10 +84,18 @@ We will coordinate our work and the issues we are working on using a single GitH
 
 This group will focus on all kind of documentation, from pipelines themselves to modules and general documentation to aid in getting started with nf-core and Nextflow. We are also planning on making introduction videos for each pipeline.
 
+  - [<i class="fab fa-slack"></i> `#hackathon-oct-2022-documentation`](https://nfcore.slack.com/archives/C041QU9SX09)
```

Comment:
Are these indented for a reason? It's breaking prettier linting..

Reply:
They pass prettier linting in my computer 🤔 but I will remove the indentation just in case